src/command_buffer.rs

```diff
@@ -121,6 +121,7 @@ impl CommandBuffer {
     }
 
     fn build(&mut self, mark: usize) -> (Entity, ReadyBuffer<'_>) {
+        self.ids.clear();
```

Comment:
previously this resulted in generating archetypes for per entity even if they had the same component types,